src/main/java/frc/robot/arm/IArmIO.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.arm;
+
+public interface IArmIO {
+
+    //provides power value from -1.0 to 1.0
+    public void setMotorPower(double power);
```

Comment:
According to this the power should be -1 to 1, but in your subsystem you are still multiplying by 12. I am guessing that this is just left over from when we were setting voltages. We should standardize converting to volts within the interface implementations.